components/Tabs/Tabs.js

```diff
@@ -1,61 +1,85 @@
+// STEP 3: Create the class. Classes aren't hoisted!
 class TabLink {
   constructor(tabElement){
-    // assign this.tabElement to the tabElement DOM reference
-    // this.tabElement;
-    
-    // Get the `data-tab` value from this.tabElement and store it here
+    // console.log(tabElement);
+
+// STEP 4: Matched the data-set values between our links and our content
+
+  // assign this.tabElement to the tabElement DOM reference
+    // this.tabElement = document.querySelector(`.card[data-tab="${link.dataset.tab}"]`);
+    this.cardsContainer = document.querySelector(`.card[data-tab="${tabElement.dataset.tab}"]`);
```

Comment:
We walked through this in our 1:1, so you already know about how to assign this.tabElement to the correct DOM reference. I know it was tricky, but make sure you read through the comments and instructions even before adding your own just so that the two don't get too confused.